docs/usage.md

```diff
@@ -3,130 +3,317 @@
+
+1. **Initiate Navigation**:
+   - Issue a command to navigate to a specific page or module. For example:
+
+    >*Navigate to the Alerts module*
+
+    or
+
+    >*Navigate to Content Hub*
+
+   - The assistant directs you to the requested module, such as Alerts, Incidents, Indicators, or in the above case &ndash; to Content Hub.
+
+2. **Access a Specific Record**:
+   - Provide details such as the record's identifier or name to access a particular record. For example:
+
+    >*Open the record with ID 12345*
+
+   - The assistant locates and displays the record for you.
+
+Adjust the navigation command to suit your needs, such as specifying a different module or record.
+
+### Review Navigation Commands
+
+- Review your navigation commands by verifying that they direct you to the correct module or record. Make adjustments if necessary to ensure accurate access.
+
+### Integrate Navigation into Workflow
+
+- Connect the navigation commands into your workflow to enhance efficiency. Utilize the commands to quickly move between modules and access the necessary records.
+
+Effective navigation within FortiSOAR is crucial for managing tasks and accessing information efficiently. By following these steps and utilizing the FortiAI assistant's navigation capabilities, users can streamline their workflows and improve productivity.
+
+>[!NOTE]
+>Familiarize yourself with the specific commands for navigation to optimize your use of the FortiAI assistant.
+
+## Providing Context
+
+Providing context within the FortiAI assistant involves loading, retrieving, fetching, accessing, or providing additional information about an *already opened* record. This functionality enhances the ability to analyze and act on data by ensuring that relevant information is readily available.
+
+### Use Case Example: Retrieving or Enriching Record Details
+
+A common use case involves providing additional details about an alert or incident that is already opened. For example, an analyst might need to access the full context of an alert to understand its background and relevance. By providing additional context, the analyst can make more informed decisions and take appropriate actions.
+
+### Providing Context Using FortiAI Assistant
+
+1. **Load Additional Information**:
+   - Issue a command to load more information about an *already opened* record. For example:
+
+    >*Provide context for the alert with ID 12345*
+
+   - The assistant retrieves and displays additional details related to the opened record.
+
+2. **Retrieve Specific Data**:
+   - Specify the type of data you need to retrieve about the record. For example:
+
+    >*Fetch the incident history for the current incident*
+
+   - The assistant provides the requested data, enhancing your understanding of the record.
+
+3. **Access Detailed Records**:
+   - Access detailed information about the record as needed. For example:
+
+    >*Access the full details of the alert for further analysis*
```

Comment:
Need to change this prompt @cs-ankit-gupta

Reply:
Done